src/route.rs

```diff
@@ -95,14 +95,13 @@ impl Route {
 
     pub(crate) fn set_unmatched_path(&mut self, index: usize) {
         let index = self.segments_index + index;
-
         let path = self.req.uri().path();
```

Comment:
can we use `path.is_empty()` instead , and add a comment noting path malformed for the empty path?

Reply:
how's that ?